swagger-spec/users/external_identities_detail.yaml

```diff
@@ -0,0 +1,96 @@
+get:
+  summary: Retrieve user's external identities
+  description: >-
+    Retrieves a JSON object of a user's external identity, currently this only includes their ORCID account.
+
+    The returned information includes the external identity's name, status and id.
+
+    #### Returns
+
+    Returns a JSON object with a `data` key containing the representation of the requested
+    external identities, if the request is successful.
+
+    If the request is unsuccessful, an `errors` key containing
+    information about the failure will be returned. Refer to the [list of error codes](#tag/Errors-and-Error-Codes)
+    to understand why this request may have failed.
+
+
+  parameters:
+  - in: path
+    type: string
+    name: user_id
+    required: true
+    description: 'The unique identifier of the user.'
+  - in: path
+    type: string
+    name: identities_id
+    required: true
+    description: "The unique identifier of a user's external identity."
+  tags:
+    - Users
+  operationId: external_identities_detail
+  responses:
+    '200':
+      description: 'OK'
+      schema:
+        $ref: definition.yaml
```

Comment:
This should reference an external identity definition. Currently the response schema implies that a fully serialized user is returned, when it's just the external identity.
![screen shot 2018-08-15 at 1 24 56 pm](https://user-images.githubusercontent.com/7913604/44162530-ac38bd80-a08e-11e8-93f5-5da59abe22db.png)
vs.
![screen shot 2018-08-15 at 1 25 08 pm](https://user-images.githubusercontent.com/7913604/44162549-afcc4480-a08e-11e8-86ba-3a8d11e5ff3b.png)